rafs/src/metadata/md_v5.rs

```diff
@@ -25,7 +25,7 @@ impl RafsSuper {
         self.meta.chunk_size = sb.block_size();
         self.meta.flags = RafsSuperFlags::from_bits(sb.flags())
             .ok_or_else(|| einval!(format!("invalid super flags {:x}", sb.flags())))?;
-        info!("rafs superblock features: {}", self.meta.flags);
+        debug!("rafs superblock features: {}", self.meta.flags);
```

Comment:
Um... I personally think it is helpful to investigate problems with these feature flags

Reply:
> Um... I personally think it is helpful to investigate problems with these feature flags

U r right. Just Remove unpacker's log